frontend/src/lib/OutputArtifactLoader.ts

```diff
@@ -474,7 +474,7 @@ async function getArtifactTypes(): Promise<ArtifactType[]> {
   return res.getArtifactTypesList();
 }
 
-function filterArtifactsByType(
+export function filterArtifactsByType(
```

Comment:
nit: move this method and getArtifactTypes to mlmdUtils?
they are very generic

Reply:
Sounds good, done.